src/main/java/com/fasterxml/jackson/core/JsonParser.java

```diff
@@ -348,6 +348,17 @@ public enum Feature {
           */
          USE_FAST_DOUBLE_PARSER(false),
 
+         /**
+          * Feature that determines whether we use the built-in {@link new BigDecimal(String)} code to parse
+          * <code>BigDecimal</code>s or if we use {@link com.fasterxml.jackson.core.io.doubleparser}
+          * instead. This feature has a similar effect on <code>BigInteger</code> parsing.
+          *<p>
+          * This setting is disabled by default for backwards compatibility.
+          *
+          * @since 2.15
+          */
+          USE_FAST_BIG_DECIMAL_PARSER(false)
```

Comment:
Since this affects `BigDecimal` and `BigInteger` how about `USE_FAST_BIG_NUMBER_PARSER`? And then mention it affects both types. I think it's fine to have just one setting, but if so name needs to be more generic.

Reply:
done